README.md

```diff
@@ -73,68 +73,10 @@ AtacWorks trains a deep neural network to learn a mapping between noisy (low cov
 
 ### 1. Training an AtacWorks model
 
-#### Input files
```

Comment:
Rewrite of line 72:
"AtacWorks trains a deep neural network to learn a mapping between noisy (low coverage/low cell count/low quality) ATAC-seq data and matching clean (high coverage, high cell count, and/or high quality) ATAC-seq data. Both noisy and clean data should be from the same cell type or tissue. Once this mapping is learned, the trained model can be applied to improve other noisy ATAC-Seq datasets. "